packages/postcss/mixin.core.js

```diff
@@ -13,17 +9,46 @@ const getAssetPath = (...args) => trimSlashes(join(...args));
 
 const cssLoaderLocalOptions = {
   modules: {
+    auto: () => true,
```

Comment:
`css-loader` enables CSS Modules only for files, whose names match `/\.module\.css$/`, when `auto` is set to `true`. From what I observed, we need it enabled for all files, so we rather pass a function into this option, that always returns `true`.

Reply:
Because `auto: true` enables CSS Modules for files, whose names match /\.module\.css$/. So we have to provide a custom matcher function, which basically enables it for every file — whatever their name is.